src/gui/QvisDBOptionsDialog.C

```diff
@@ -129,7 +137,12 @@ QvisDBOptionsDialog::QvisDBOptionsDialog(DBOptionsAttributes *dbatts,
           case DBOptionsAttributes::MultiLineString:
             { // new scope
             txt = atts->GetMultiLineString(name).c_str();
-            QTextEdit *textEdit = new QTextEdit(txt, this);
+            QPlainTextEdit *textEdit = new QPlainTextEdit(txt, this);
+            // this prevents users from typing tabs in the editor.
+            textEdit->setTabChangesFocus(true);
+            // use a fixed width font
+            QFont f("Courier", 10);
+            textEdit->setFont(f);
             QLabel *label = new QLabel(tr(name.c_str()), this);
             label->setAlignment(Qt::AlignTop);
```

Comment:
I don't know Qt alignment options/syntax but is *left* default? If not, do we need `setAlignment(Qt::AlignTop|Qt::AlignLeft);`

Reply:
Qt docs on QLabel alignment:
`By default, the contents of the label are left-aligned and vertically-centered.`